Integrations/CounterCraft/CounterCraft.yml

```diff
@@ -0,0 +1,420 @@
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: countercraft-list-campaigns
+    arguments:
+    - name: name
+      description: Campaign Name
+    outputs:
+    - contextPath: CounterCraft.Campaign.ID
+      description: Campaign ID
+      type: number
+    - contextPath: CounterCraft.Campaign.Name
+      description: Campaign Name
+      type: string
+    - contextPath: CounterCraft.Campaign.Description
+      description: Campaign Description
+      type: string
+    - contextPath: CounterCraft.Campaign.StatusCode
+      description: Campaign Status
+      type: string
+    description: 'List all deception campaigns'
+  - name: countercraft-list-hosts
+    arguments:
+    - name: campaign_id
+      description: Campaign ID
+    outputs:
+    - contextPath: CounterCraft.Host.ID
+      description: Host Id
+      type: number
+    - contextPath: CounterCraft.Host.Name
+      description: Host Name
+      type: string
+    - contextPath: CounterCraft.Host.Description
+      description: Host Description
+      type: string
+    - contextPath: CounterCraft.Host.StatusCode
+      description: Host Status
+      type: string
+    - contextPath: CounterCraft.Host.TypeCode
+      description: Host Type
+      type: string
+    description: Lists all deception hosts
+  - name: countercraft-list-services
+    arguments:
+    - name: host_id
+      description: Host Id
+    outputs:
+    - contextPath: CounterCraft.Service.ID
+      description: Service ID
+      type: number
+    - contextPath: CounterCraft.Service.Name
+      description: Service Name
+      type: string
+    - contextPath: CounterCraft.Service.Description
+      description: Service.Description
+      type: string
+    - contextPath: CounterCraft.Service.StatusCode
+      description: Service Status
+      type: string
+    - contextPath: CounterCraft.Service.TypeCode
+      description: Service Type
+      type: string
+    description: 'List services currently deployed on deception hosts'
+  - name: countercraft-list-breadcrumbs
+    arguments:
+    - name: campaign_id
+      description: Campaign ID
+    outputs:
+    - contextPath: CounterCraft.Breadcrumb.ID
+      description: Breadcrumb ID
+      type: number
+    - contextPath: CounterCraft.Breadcrumb.Name
+      description: Breadcrumb Name
+      type: string
+    - contextPath: CounterCraft.Breadcrumb.Description
+      description: Breadcrumb Description
+      type: string
+    - contextPath: CounterCraft.Breadcrumb.StatusCode
+      description: Breadcrumb Status
+      type: string
+    - contextPath: CounterCraft.Breadcrumb.TypeCode
+      description: Breadcrumb Type
+      type: string
+    description: List breadcrumbs in a campaign
+  - name: countercraft-get-object
+    arguments:
+    - name: value
+      required: true
+      description: Object value
```

Comment:
please add an example for a value, or how can it be retrieved

Reply:
Same than above. If I have added an example in the documentation, do I need to add it to the description field?

Reply:
Done